### DISCUSSION TITLE
Ability to Use Flows in a Node.js App Like a Chain

### DISCUSSION THREAD

**Discussion by cantaspinar** [FlowiseAI/Flowise#391] [Ideas]
Right now, we can use Flowise as an API or to embed it on a website. It would be really usefull if we could just export a flow and load it in a Node.js application to be able to use it in different environments. One example can be the firebase functions. I could create a flow, export it, write a firebase function and use the exported flow in my function like any chain.

**Comment by mattmattkim** [comment]
I recently discovered Flowise and I'm adding it to my Firebase project. My workaround is to deploy a Cloud Function that calls the Flowise endpoint. 

Using Flowise as an "LLM Agent as a Service" is a new paradigm for me to wrap my head around and curious to hear thoughts from the community.
